skimage/segmentation/morphsnakes.py

```diff
@@ -0,0 +1,440 @@
+    is required that the inside of the object looks different on average than
+    the outside (i.e., the inner area of the object should be darker or lighter
+    than the outer area on average).
+
+    Parameters
+    ----------
+    image : (M, N) array
+        Grayscale image to be segmented.
+    iterations : uint
+        Number of iterations to run
+    init_level_set : str or (M, N) array
+        Initial level set. If an array is given, it will be binarized and used
+        as the initial level set. If a string is given, it defines the method
+        to generate a reasonable initial level set with the shape of the
+        `image`. Accepted values are 'checkerboard' and 'circle'. See the
+        documentation of `checkerboard_level_set` and `circle_level_set`
+        respectively for details about how these level sets are created.
+    smoothing : uint, optional
+        Number of times the smoothing operator is applied per iteration.
+        Reasonable values are around 1-4. Larger values lead to smoother
+        segmentations.
+    lambda1 : float, optional
+        Weight parameter for the outer region. If `lambda1` is larger than
+        `lambda2`, the outer region will contain a larger range of values than
+        the inner region.
+    lambda2 : float, optional
+        Weight parameter for the inner region. If `lambda2` is larger than
+        `lambda1`, the inner region will contain a larger range of values than
+        the outer region.
+    iter_callback : function, optional
+        If given, this function is called once per iteration with the current
+        level set as the only argument. This is useful for debugging or for
+        plotting intermediate results during the evolution.
+
+    Returns
+    -------
+    out : (M, N) array
+        Final segmentation (i.e., the final level set)
+
+    See also
+    --------
+    circle_level_set, checkerboard_level_set
+
+    Notes
+    -----
+
+    This is a version of the Chan-Vese algorithm that uses morphological
+    operators instead of solving a partial differential equation (PDE) for the
+    evolution of the contour. The set of morphological operators used in this
+    algorithm are proved to be infinitesimally equivalent to the Chan-Vese PDE
+    (see [1]_). However, morphological operators are do not suffer from the
+    numerical stability issues typically found in PDEs (it is not necessary to
+    find the right time step for the evolution), and are computationally
+    faster.
+
+    The algorithm and its theoretical derivation are described in [1]_.
+
+    References
+    ----------
+    .. [1] A Morphological Approach to Curvature-based Evolution of Curves and
+           Surfaces, Pablo Márquez-Neila, Luis Baumela, Luis Álvarez. In IEEE
+           Transactions on Pattern Analysis and Machine Intelligence (PAMI),
+           2014, DOI 10.1109/TPAMI.2013.106
+    """
+
+    # TODO: Make it work with "color" (or multi-channel) images.
```

Comment:
Is this still on the horizon?

Reply:
In general, we don't have TODOs in our code (is this true? I think it's true). Instead, remove this line, and raise an issue on GitHub describing the feature request, once this is merged.

Reply:
It is on the horizon, but not in the short term. I'll remove the TODO line.